### PR TITLE
Add screnshot to readme

### PR DESCRIPTION
**Pull request by AugustasV**
#167 Adding screenshot of zoom plugin to repo README.md file.

Signed-off-by: AugustasV <reg1nt1z@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 [![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-zoom)](https://github.com/mattermost/mattermost-plugin-zoom/releases/latest)
 [![HW](https://img.shields.io/github/issues/mattermost/mattermost-plugin-zoom/Up%20For%20Grabs?color=dark%20green&label=Help%20Wanted)](https://github.com/mattermost/mattermost-plugin-zoom/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Up+For+Grabs%22+label%3A%22Help+Wanted%22)
 
+![Mattermost Plugin dashboard](https://raw.githubusercontent.com/mattermost/mattermost-plugin-zoom/e1d5480691d4c184d028ea4028eae7421810729b/public/setup_flow_images/oauth_scope_summary.png)
```
Comment (justinegeffen) [inline README.md:8]: This image needs to be stored in the `assets` folder in this repo. The reference will then look something like this: ``. Let me know if you have any trouble with that! :)

**Comment by AugustasV** [comment]
@mickmister

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
The image isn't displaying in the preview because it hasn't been merged. I'll need to check this in prod.

**Comment by justinegeffen** [comment]
ping for this, @larkox. :)

**Comment by larkox** [comment]
@AugustasV Sorry for the confusion. My idea was more in the line of a screenshot like this one (used on older versions of the readme), but updated with the latest UI.
Screenshot from 2022-03-16 13-40-49

Also, with the work @mickmister is doing with the App Bar, this UI may soon change again.

@mickmister Do you think we should close this PR and wait for the App Bar to be in place to add the screenshot? Or do you see value on adding this now, and update it later?

**Comment by larkox** [comment]
Thanks for confirming. I am really sorry @AugustasV for all the confusion. Thank you very much for contributing, and we hope you contribute in other parts of the product.

**Pull request closed without merging**